low-level/files/opt/seagate/sspl/conf/setup.yaml

```diff
@@ -44,4 +44,4 @@ sspl:
          args: null
 
 support_bundle:
-    - /opt/seagate/cortx/sspl/bin/sspl_bundle_generate
+    - /usr/bin/sspl_bundle_generate
```

Comment:
why these path changes? dont change until there is an ask or surely required reason

Reply:
For binary code we have created this symbolic link in /opt/seagate/cortx/sspl/bin/ but now we have created a symbolic link in /usr/bin/sspl_bundle_generate for easier access to this script.